pulp-worker/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM kong/pulp-core:0.0.12
+ARG FROM_ORG
+ARG FROM_TAG=latest
+FROM ${FROM_ORG}pulp-core:$FROM_TAG
```

Comment:
dependatbot and docker best practices prefer this to be explicitely tagged but I'm comfortable with it not being tagged personally

Reply:
Good catch.  They're tagged explicitly with a version number if the build argument is provided - which is what the Makefile does in the CI runs (Makefile#L32).  There just needs to be a default value here so local execution without setting variables still works. Otherwise the trailing colon would break running just a `docker build` (the makefile also defaults to latest if no TAG is set, so this would always work with `make images` even without a default).  So the use of "latest" is really just a last-resort failsafe if someone does a local build of one container; in that case I think they would almost certainly want latest (probably from their own machine). :shrug:

Any new version of the pulp-core container these sub-containers depend upon will only exist after this container is updated, so we should be safe with Dependabot on these.  The pulp-core container itself uses Fedora with `:33` right now, so that's in good shape too. :)